components/Feed/Caption.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+import { useEffect, useState } from "react";
+
+export default function Caption({ caption }: { caption: string | null }) {
```

Comment:
Maybe we can just require a caption, to simplify the code internally, then set it on the wrapper component:
```tsx
<Caption caption={video.caption || ""} />
```